iothub/device/devdoc/retrypolicy.md

```diff
@@ -12,3 +12,26 @@ __Versions below 1.19.0 only:__ If the service is responding with a throttling e
 `new ExponentialBackoff(RetryCount, TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(60), TimeSpan.FromSeconds(5))`
 
 The retry mechanism will stop after `DefaultOperationTimeoutInMilliseconds` which is currently set at 4 minutes.
+
+The SDK also provides a `NoRetry` implementation of the `IRetryPolicy`. This can be used if you do not want the client to retry after getting disconnected.
+
+You can also supply your custom retry policy to the `DeviceClient` by implementing the `IRetryPolicy` interface.
+
+``` C#
+public class MyCustomRetryPolicy : IRetryPolicy
+{
+    /// <summary>
+    /// Returns true if, based on the parameters the operation should be retried.
```

Comment:
Missing a comma. Should be:

Returns true if, based on the parameters, the operation should be retried.